tests/everest/test_res_initialization.py

```diff
@@ -102,7 +102,8 @@ def test_everest_to_ert_queue_config(config, config_class):
     assert ert_config.queue_config.queue_options == config_class(**config)
 
 
-def test_everest_to_ert_controls():
+def test_everest_to_ert_controls(tmp_path, monkeypatch):
```

Comment:
We could merge this, but this should not really create any files, so would be interesting to see what side effect it has, and then fix that.